Clients/GUI/CustomValidationAttributes/BirthdayAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using GUI.ViewModels;
+
+namespace GUI.CustomValidationAttributes
+{
+    public class BirthdayAttribute : ValidationAttribute
+    {
+        public string GetErrorMessageTooYoung() =>
+            $"You are to young.";
```

Comment:
What's the reason of `$` here? I don't see interpolation

Reply:
And may be it could be a property, but ok

Reply:
fixed